java/serve/src/main/java/io/ray/serve/serializer/Hessian2Seserializer.java

```diff
@@ -0,0 +1,37 @@
+package io.ray.serve.serializer;
+
+import com.caucho.hessian.io.Hessian2Input;
```

Comment:
Why do we use Hessian as the serialization format here? If it's expected to be used cross language we should use messagepack, if not is Hessian a standard serialization protocol for Java? If so does Ray core use it?

Reply:
Hesssian's compatibility is better when adding new properties to the object. But MsgPack performance better in cross-language scenarios. I think MsgPack is the right choice here.